buildcatrust/certstore_output.py

```diff
@@ -29,6 +29,27 @@ def output(self, cert: Optional[types.Certificate], trust: types.Trust) -> None:
                 file=self.fp,
             )
 
+
+class StandardCertStoreOutput(CertStoreOutput):
+    """
+    PEM standard abiding output when the OpenSSL specific format
+    with trust rules is incompatible with your software.
+    """
+
+    def __init__(self, fp: TextIO):
+        self.fp = fp
+
+    def output(self, cert: Optional[types.Certificate], trust: types.Trust) -> None:
+        super().output(cert, trust)
```

Comment:
This doesn't do anything different to the parent class; maybe just delete everything here apart from the docstring?

Reply:
Oops, thank you for taking care of it.